### DISCUSSION TITLE
Code not showing as expected

### DISCUSSION THREAD

**Discussion by EPrD** [community/community#119850] [Repositories]
### Select Topic Area

Question

### Body

I have updated some code but not all of it is showing in the repository, yet if I do a code comparison, the change is there. Other changes in the same code file are shown correctly, it looks just like one line is affected. I have checked that I have the correct branch and the commit code is matching.

I am not sure what is going on. Has anyone experienced anything similar?

**Comment by ebndev** [comment]
Hey @EPrD!

Thanks for posting in the community.

Did you push you changes?

**Reply by EPrD** [reply]
I have pushed changes and downloaded successfully onto another device so all is functioning ok, but on Github the old code is there unless I examine code changes (which incidentally shows an update from a later change than the code shows so it's not reflecting a couple of updates). It's just one line but it is a mystery and I noticed it because it contains text of my version number of the code ('v0.1.0.0416b')

**Reply by EPrD** [reply]
I have discovered the issue - I had a problem with my local files and so there is no issue with Github. Thank you for your help.
